students/olszewski_bartosz/lesson_03_functions/collatz.py

```diff
@@ -0,0 +1,15 @@
+def collatz(number):
+    if number % 2 == 0:
+        print(number // 2)
+        return number // 2
+    elif number % 2 == 1:
```

Comment:
not necessary elif statement. When using if <-> elif always ends conditions with else (if <-> elif  <-> else) even if you think that it will never reach else.